nimble/host/include/host/ble_hs.h

```diff
@@ -184,6 +184,22 @@ extern "C" {
 
 /** CSRK distibution and LinkKey are not supported */
 
+/**
+ * @}
+ */
+
+/**
+ * @brief LE deprecate warning
+ *
+ * @{
+ */
+
+#if MYNEWT_VAL(BLE_SUPPRESS_DEPRECATE_WARN)
+#define NIMBLE_DEPRECATED(_msg) __attribute__((deprecated("Deprecated. Should not be used. " _msg)))
```

Comment:
This should be the other way around... but I really think that this feature (warn on deprecate) should be opt in, ie  name it BLE_WARN_ON_DEPRECATE and default to 0 and warn only if it is set to 1

Reply:
It's default value is set to 0, if someone wants to opt in they can set it to 1.

Reply:
BLE_SUPPRESS_DEPRECATE_WARN:
        description: >
            This suppresses deprecated warning
        value: 0


Enabling warning should be opt-int, not disabling those  ie:

BLE_WARN_ON_DEPRECATE:
        description: >
            This enabling warning on used deprecated API.
        value: 0